Issue 166

johnpj opened the pull request:
Fixes #166

jonespm commented:
For some reason your PR's still have a lot of old commits. I think you still need to pull a update from your master. It might not be pulling because you have older commits that are conflicting. You can 

https://help.github.com/desktop/guides/contributing-to-projects/syncing-your-branch/

I'm not sure how the UI works but you might be able to just reset or stash whatever changes you have. (Of course easiest would be to just delete and reclone). On the command line it's usually either deleting the branch and checking it out again or `git reset --hard`

zqian commented:
Please do not checkin dashboard/static/.DS_Store file

zqian commented:
Can we make screenshots based on more recent visualizations? There has been a lot of changes in all three views.

jonespm commented:
Yeah, I added the .DS_Store to the .gitignore list. I'm not sure why the other changes are still here though. Probably a local conflict.

johnpj commented:
Apologies. I deleted and re-cloned my local branch.

On Tue, Oct 2, 2018 at 9:53 PM Code Hugger (Matthew Jones) <
notifications@github.com> wrote:

> Yeah, I added the .DS_Store to the .gitignore list. I'm not sure why the
> other changes are still here though. Probably a local conflict.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tl-its-umich-edu/student-dashboard-django/pull/167#issuecomment-426486885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFX1Cizhwkq_HcNOqgkSMKKH5vFI6si3ks5uhBidgaJpZM4XE_hs>
> .
>


-- 
John Johnston
IT Program Manager
University of Michigan
<https://www.facebook.com/mr.john.johnston>
<https://twitter.com/johnpjohnston>
<https://www.linkedin.com/in/john-johnston-9479627/>
<https://universityofmichigan.facebook.com/profile.php?id=100024588590723>

jonespm commented:
I'm going to close this one, can you put up another PR?
